README.md

```diff
@@ -1,47 +1,39 @@
-# Netlify open-api
+# open-api
+[![Build Status](https://travis-ci.org/netlify/open-api.svg?branch=master)](https://travis-ci.org/netlify/open-api) [![Build status](https://ci.appveyor.com/api/projects/status/qrmvxk957ou2yrd9/branch/master?svg=true)](https://ci.appveyor.com/project/netlify/open-api/branch/master)
```

Comment:
can we have a new line for each status? I _think_ it still renders the same way

Reply:
Fixing